From 4104b73dbec432ae264d4f05beaaba34796ae62b Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 18 Oct 2022 23:01:56 +0100
Subject: [PATCH 3/7] gnu: pytest-astropy-header: Update to 0.2.2

* gnu/packages/python-check.scm (python-pytest-astropy-header): Update
  to 0.2.2
---
 gnu/packages/python-check.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/python-check.scm b/gnu/packages/python-check.scm
index e3b8ede9f0..e7828990c1 100644
--- a/gnu/packages/python-check.scm
+++ b/gnu/packages/python-check.scm
@@ -432,13 +432,13 @@ (define-public python-pytest-ordering
 (define-public python-pytest-astropy-header
 (package
   (name "python-pytest-astropy-header")
-  (version "0.1.2")
+  (version "0.2.2")
   (source
     (origin
       (method url-fetch)
       (uri (pypi-uri "pytest-astropy-header" version))
       (sha256
-        (base32 "1y87agr324p6x5gvhziymxjlw54pyn4gqnd49papbl941djpkp5g"))))
+        (base32 "046v4arinv8b5jz05pvhnc0n1aqqndwvhlsl635ahxabr40i32bp"))))
   (build-system python-build-system)
   (native-inputs
    (list python-pytest python-setuptools-scm))
-- 
2.37.3

